var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -44,7 +45,7 @@ class VtkM(CMakePackage, CudaPackage):
     variant("64bitids", default=False,
             description="enable 64 bits ids")
 
-    depends_on("cmake")
+    depends_on("cmake@3.12:")         # CMake >= 3.12
```

Comment:
```suggestion
    depends_on("cmake@3.12:", type="build")         # CMake >= 3.12
```
I assume this dependency is only needed at build-time